garage/replay_buffer/base.py

```diff
@@ -49,15 +50,20 @@ def sample(self, batch_size):
         """Sample a transition of batch_size."""
         raise NotImplementedError
 
-    def add_transitions(self, **kwargs):
+    def add_transition(self, **kwargs):
         """Add one transition into the replay buffer."""
+        transition = {k: [v] for k, v in kwargs.items()}
+        self.add_transitions(**transition)
+
+    def add_transitions(self, **kwargs):
```

Comment:
please provide docstrings for this file. if they live in subclasses, please move them to this base class.

is this file well-tested?